plugins/workspace-minimap/src/index.ts

```diff
@@ -0,0 +1,38 @@
+/**
+ * @license
+ * Copyright 2023 Google LLC
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+
+/**
+ * @fileoverview A minimap is a miniature version of your blocks that
+ * appears on top of your main workspace. This gives you an overview
+ * of what your code looks like, and how it is organized.
+ */
+import * as Blockly from 'blockly/core';
+
+/**
+ * A minimap is a miniature version of your blocks that appears on
+ * top of your main workspace. This gives you an overview of what
+ * your code looks like, and how it is organized.
+ */
+export class Minimap {
+  /** The workspace. */
+  protected workspace: Blockly.WorkspaceSvg;
+  /**
+   * Constructor for ...
+   * @param workspace The workspace that the plugin will
+   *     be added to.
+   */
+  constructor(workspace: Blockly.WorkspaceSvg) {
```

Comment:
For simple cases where you're just directly assigning a constructor arg to an instance variable, you can use [parameter properties](https://www.typescriptlang.org/docs/handbook/2/classes.html#parameter-properties) to simplify things a bit.

Reply:
Discussed offline that this will be resolved in a followup.